### ISSUE TITLE
[QuickOpen] Style input bar

### ISSUE THREAD

**Issue by jasonLaster** [devtools-html/debugger.html#5906]
Here are some tweaks we would like to make to the quick open modal

- [x] bar should be larger
- [x] bar should have a blue focus ring
- [ ] loading should appear in the right secton of the bar

![](https://user-images.githubusercontent.com/5976612/38178122-8c327e0e-35d1-11e8-8200-2a3cd47d5e7b.png)

**Comment by darkwing** [comment]
@violasong Do we have a style guide for font sizes?

**Comment by dimitardanailov** [comment]
@jasonLaster and @darkwing 

Greetings :) 

I’d like to work on this ticket

**Comment by Sekhmet** [comment]
@dimitardanailov 

You can use `/claim` to claim this issue.

**Comment by violasong** [comment]
Font sizes: Text inside the input field should be the equivalent of 13px; everything else - 12px.

Colors: Gray 90 for file name, Gray 50 for path/placeholder text, Gray 40 for "Loading..."

No divider lines needed between rows, and less spacing between rows would be great. (Should look similar to the main Firefox URL bar menu)

Thanks for working on this Dimitar!

**Comment by jasonLaster** [comment]
closing as a dupe of #6180
